src/components/molecules/SecretPasswordForm/SecretPasswordForm.tsx

```diff
@@ -1,34 +1,51 @@
 import React, { useCallback, useState } from "react";
-import { useHistory } from "react-router-dom";
 
 import { checkAccess } from "api/auth";
 
-import { venueEntranceUrl } from "utils/url";
 import { setLocalStorageToken } from "utils/localStorage";
 import { isTruthy } from "utils/types";
 
 import { useVenueId } from "hooks/useVenueId";
+import { useUser } from "hooks/useUser";
 
 import "./SecretPasswordForm.scss";
 
-const SecretPasswordForm = ({ buttonText = "Join the party" }) => {
-  const history = useHistory();
+interface SecretPasswordFormProps {
```

Comment:
Is there a particular reason to export something that's not being used from the outside of this file?

Reply:
Mostly 'past frustration'/annoyance with libraries that 'lock away' their types, where I might want to use them. I feel like the type of the component's props forms part of its 'public contract', so it just "feels more right" that they are exported along with it.

At the end of the day, if they aren't, it's not going to break things, and at worst it may just require additional changes if something wants to use them in the future. It's more of a personal preference around 'doing things well', as I tend to think from the perspective of what I would expect/want to see from a public library I was consuming/etc.

Reply:
Actually, it is often also really useful when it comes to writing tests/etc, and being able to reuse things like that, without having to go and also change the source files when you are.

Reply:
(captured this in https://github.com/sparkletown/sparkle/pull/1062#issuecomment-776407944)

Reply:
Makes sense I'll export the interfaces as well. I'll consider this as a convention and do it from now on.

Reply:
Great, thank you for that :)